lib/rules/color-named/index.js

```diff
@@ -113,8 +113,8 @@ const rule = (primary, secondaryOptions) => {
 					const normalizedFunctionString = valueParser.stringify(node).replace(/\s+/g, '');
 					let namedColor;
 
-					for (let i = 0, l = namedColors.length; i < l; i++) {
-						namedColor = namedColors[i];
+					for (const namedColor_ of namedColors) {
+						namedColor = namedColor_;
```

Comment:
Is this right?

Reply:
TBH this baffled me too. It seems that `namedColor` is shared between the two loops. Maybe they could/should be combined later?

Reply:
I think this PR is good timing to combine them! 👍🏼

Reply:
Feel free to push here :) Or someone could do it in another PR later

Reply:
Fixed! 9ae83efa